src/components/Icon/Icon.tsx

```diff
@@ -1,46 +1,69 @@
     </svg>
   ),
-  "logout": () => (
+  "support": () => (
     <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
-      <path d="M9.52683 5H6C5.44772 5 5 5.44772 5 6V18C5 18.5523 5.44771 19 6 19L9.52683 19M9.43783 12.1872L19.1872 12.1872M19.1872 12.1872L15.9374 15.437M19.1872 12.1872L15.9374 8.93737" stroke="#333333" strokeLinecap="round" strokeLinejoin="round" />
+      <path d="M9.38097 9.29996C9.59256 8.69847 10.0102 8.19126 10.5599 7.86818C11.1096 7.54511 11.756 7.42701 12.3844 7.5348C13.0129 7.6426 13.5829 7.96933 13.9935 8.45714C14.4042 8.94494 14.6289 9.56233 14.628 10.2C14.628 12 11.928 12.9 11.928 12.9M12 16.5H12.009M12 21C16.9705 21 21 16.9705 21 12C21 7.02943 16.9705 3 12 3C7.02943 3 3 7.02943 3 12C3 16.9705 7.02943 21 12 21Z" stroke="#333333" stroke-linecap="round" stroke-linejoin="round" />
     </svg>
   ),
-  "profile": () => (
+  "user": () => (
     <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
       <path d="M12.5 13.6C15.1924 13.6 17.375 11.451 17.375 8.8C17.375 6.14903 15.1924 4 12.5 4C9.80761 4 7.625 6.14903 7.625 8.8C7.625 11.451 9.80761 13.6 12.5 13.6ZM12.5 13.6C16.0899 13.6 19 16.4654 19 20M12.5 13.6C8.91015 13.6 6 16.4654 6 20" stroke="#333333" strokeLinecap="round" strokeLinejoin="round" />
     </svg>
-  )
+  ),
 } as const;
 
 export type IconNames = keyof typeof Icons;
 
 interface IconProps {
   name: IconNames;
   hasBadge?: boolean;
-  classWrapName?: string;
+  className?: string;
 }
 
 export const Icon = ({
   ...props
 }: IconProps) => {
-  const { name, hasBadge, classWrapName } = props;
+  const { name, hasBadge, className } = props;
 
   return (
-    <div className={classWrapName}>
+    <span className={className}>
       {Icons[name](hasBadge)}
-    </div>
+    </span>
   )
 };
```

Comment:
I wonder if you can simplify the `Icon` component where you can directly use the imported icon components.

```
import React from 'react';

// Import the individual icon components
import DashboardIcon from './components/icons/DashboardIcon';
import LogoutIcon from './components/icons/LogoutIcon';
// ...and other icon components

// Create a mapping of icon names to their corresponding components
const iconComponents = {
  dashboard: DashboardIcon,
  logout: LogoutIcon,
  // ...and other icon components
} as const;

export type IconNames = keyof typeof iconComponents;

interface IconProps {
  name: IconNames;
  hasBadge?: boolean;
  className?: string;
}

export const Icon = ({ name, hasBadge, className }: IconProps) => {
  const IconComponent = iconComponents[name];

  return (
    <span className={className}>
      <IconComponent />
    </span>
  );
};
```


DashboardIcon.js:

```
import React from ‘react’;
const DashboardIcon = () => (
  <svg xmlns=“http://www.w3.org/2000/svg” width=“20" height=“20” viewBox=“0 0 20 20" fill=“none”>
    {/* SVG paths */}
  </svg>
);
export default DashboardIcon;
```

LogoutIcon.js:

```
import React from 'react';

const LogoutIcon = () => (
  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
    {/* SVG paths */}
  </svg>
);

export default LogoutIcon;
```

Reply:
I think this is a good idea.

Reply:
If icons are abstracted out, it can be more maintainable, too. Allows future devs to add icon types.

Reply:
I think it makes the code cleaner, but I don't think it needs to be in this PR.

Reply:
I agree.  Doesn't need to be in this PR.  Especially because we will research implementing this with sass and css classes in the next sprint.  Could turn out to be a better approach, in which case we would refactor or remove this component.